flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/parameters/AbstractKubernetesParameters.java

```diff
@@ -159,16 +160,16 @@ public boolean hasLog4j() {
 
 	@Override
 	public Optional<String> getLocalHadoopConfigurationDirectory() {
-		final String[] possibleHadoopConfPaths = new String[] {
-			System.getenv(Constants.ENV_HADOOP_CONF_DIR),
-			System.getenv(Constants.ENV_HADOOP_HOME) + "/etc/hadoop", // hadoop 2.2
-			System.getenv(Constants.ENV_HADOOP_HOME) + "/conf"
```

Comment:
Why is it ok to remove this line here?

Reply:
At the very beginning, I think that we have two reasons to remove the hadoop1 config directory here.
* It never works in the old code path since `System.getenv(Constants.ENV_HADOOP_HOME) + "/etc/hadoop"` could not be null or empty.
* Maybe we do not need to support Hadoop 1.x in Flink.

However, to improve the usability, I agree with you that there's no harm to also support hadoop1 conf directory here.

Reply:
Ah ok, so `System.getenv(Constants.ENV_HADOOP_HOME) + "/conf"` is only relevant for Hadoop 1.x? In this case, I am fine with removing it since we no longer test with Hadoop 1.x.